### ISSUE TITLE
Grafana prometheus-alertmanager-datasource plugin failed to download

### ISSUE THREAD

**Issue by bentsi** [scylladb/scylla-monitoring#410]
Sometimes `start-all.sh` fails since Grafana didn't succeed to download on of the plug-ins,
so some retry mechanism is needed here
```
Oct 07 11:59:07 ip-172-30-0-180.ec2.internal dockerd-current[2267]: installing grafana-piechart-panel @ 1.3.3
Oct 07 11:59:07 ip-172-30-0-180.ec2.internal dockerd-current[2267]: from url: https://grafana.com/api/plugins/grafana-piechart-panel/versions/1.3.3/download
Oct 07 11:59:07 ip-172-30-0-180.ec2.internal dockerd-current[2267]: into: /var/lib/grafana/plugins
Oct 07 11:59:07 ip-172-30-0-180.ec2.internal dockerd-current[2267]: 
Oct 07 11:59:15 ip-172-30-0-180.ec2.internal dockerd-current[2267]: ✔ Installed grafana-piechart-panel successfully 
Oct 07 11:59:15 ip-172-30-0-180.ec2.internal dockerd-current[2267]: 
Oct 07 11:59:15 ip-172-30-0-180.ec2.internal dockerd-current[2267]: Restart grafana after installing plugins . <service grafana-server restart>
Oct 07 11:59:15 ip-172-30-0-180.ec2.internal dockerd-current[2267]: 
Oct 07 11:59:25 ip-172-30-0-180.ec2.internal dockerd-current[2267]: Failed to send requesterrorGet https://grafana.com/api/plugins/repo/camptocamp-prometheus-alertmanager-datasource: net/http: request canceled (Client.Timeout exceeded while awaiting headers)
Oct 07 11:59:25 ip-172-30-0-180.ec2.internal dockerd-current[2267]: Error: ✗ Failed to send request. error: Get https://grafana.com/api/plugins/repo/camptocamp-prometheus-alertmanager-datasource: net/http: request canceled (Client.Timeout exceeded while awaiting headers)
Oct 07 11:59:25 ip-172-30-0-180.ec2.internal dockerd-current[2267]: 
Oct 07 11:59:25 ip-172-30-0-180.ec2.internal dockerd-current[2267]: NAME:
Oct 07 11:59:25 ip-172-30-0-180.ec2.internal dockerd-current[2267]:    Grafana cli plugins install - install <plugin id> <plugin version (optional)>
Oct 07 11:59:25 ip-172-30-0-180.ec2.internal dockerd-current[2267]: 
Oct 07 11:59:25 ip-172-30-0-180.ec2.internal dockerd-current[2267]: USAGE:
Oct 07 11:59:25 ip-172-30-0-180.ec2.internal dockerd-current[2267]:    Grafana cli plugins install [arguments...]
Oct 07 11:59:25 ip-172-30-0-180.ec2.internal dockerd-current[2267]: time="2018-10-07T11:59:25.139144167Z" level=error msg="containerd: deleting container" error="exit status 1: \"container 0f3284e54ee7186e25faa8bf9a51ea1546a79f2a555b9d268190726c8a165f3c does not exist\\none or more of the container deletions failed\\n\""
```
cc: @roydahan

**Comment by tzach** [comment]
related to https://github.com/scylladb/scylla-grafana-monitoring/issues/222

**Comment by bentsi** [comment]
Is it fixed?

**Comment by amnonh** [comment]
Fixed in 2.1
